### ISSUE TITLE
Time

### ISSUE THREAD

**Issue by b121351** [Engineering-Course/LIP_JPPNet#10]
What is the expected runtime for pose and parsing? When I'm testing with multiple images,  >4secs is taken for the first followed by 1sec(apprx.) for the remaining images excluding the load time.

**Comment by Engineering-Course** [comment]
Actually, the inference time depends on your input size. It will be slower for the multi-scale inference.

**Comment by b121351** [comment]
Yes! I agree. But the image which is taking taking around 1 sec when placed at non-first position is taking 4-6 secs when placed at first position. The same thing applies to all the images. That's the issue I'm talking about.

**Comment by anantaarora** [comment]
Hi @Engineering-Course,
How to load the model separately ? Please help.
